package.json

```diff
@@ -257,6 +267,16 @@
           "command": "quarkusTools.deployToOpenShift.short",
           "when": "quarkusProjectExistsOrLightWeight && resourceExtname == .java",
           "group": "navigation@10"
+        },
+        {
+          "command": "qute.validation.enabled.toggle.on",
```

Comment:
After you take into account the file's validation state, I think it should be possible to merge these 2 commands into a single toggle command, along with registering a single command.

If the user toggles the validation, you'll first access the current validation state of the file (using the command Angelo has provided). Once you know that, you'll know which way to change the state, and that can all be done in one command.

Reply:
Is there a way to assign 2 icons to a single command? Is there also a way from hiding the command from the command palette?

Reply:
Nope, nevermind, can't be done. A menu is tied to a command, not to an icon.